specification/communication/data-plane/JobRouter/preview/2021-10-20-preview2/communicationservicejobrouter.json

```diff
@@ -0,0 +1,3635 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Communication Routing Service",
+    "description": "Azure Communication Routing Service",
+    "version": "2021-10-20-preview2"
+  },
+  "paths": {
+    "/routing/classificationPolicies": {
+      "post": {
+        "tags": [
+          "ClassificationPolicies"
+        ],
+        "summary": "Creates a new classification policy.",
+        "operationId": "JobRouter_CreateClassificationPolicyV2",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "header",
```

Comment:
Every parameter should have a description.  Please add a description here and for any other parameter where one is missing.

Reply:
Tracked using [issue](https://skype.visualstudio.com/SPOOL/_workitems/edit/2788061)